statistic-labels/index.html

```diff
@@ -327,6 +327,7 @@ <h3 id="printBasinTypeHeader">Basin Characteristic Types</h3>
 
 			<p class="showStatTypes">
```

Comment:
Consider changing this div so that it takes up more width of the page? Right now it awkwardly takes up only half the width of the page. 
![image](https://user-images.githubusercontent.com/40237491/147693116-3f019c25-4b04-40e6-910f-16c66e55f62f.png)